wezterm-gui/src/termwindow/mouseevent.rs

```diff
@@ -105,8 +116,24 @@ impl super::TermWindow {
                 let button = mouse_press_to_tmb(press);
 
                 let click = match self.last_mouse_click.take() {
-                    None => LastMouseClick::new(button, (x, y)),
-                    Some(click) => click.add(button, (x, y)),
+                    None => LastMouseClick::new(
```

Comment:
nit: you could:

```
let click_position = ClickPosition {
   ...
};
```

ahead of this block, and then use pass that down to the `new` and `add` calls to save a little bit of repetition.